src/Pos/Wallet/Web/ClientTypes.hs

```diff
@@ -272,6 +272,7 @@ data CAddress = CAddress
     { cadId     :: !(CId Addr)
     , cadAmount :: !CCoin
     , cadIsUsed :: !Bool
+    , cadIsChange :: !Bool -- ^ Is this a change address
```

Comment:
we have formatter that, when applied, formats the code. Mine is applied on every file save in the project. Ask in serokell general channel for details how to set it up